recipes/searchgui/meta.yaml

```diff
@@ -1,29 +1,33 @@
 {% set name = "SearchGUI" %}
-{% set version = "3.3.10" %}
+{% set version = "4.0.1_SNAPSHOT" %}
 # Do not forget to update the version string in the searchgui.py file
 
 about:
   home: https://github.com/compomics/searchgui
-  license: Apache License, Version 2.0
-  summary: |
+  license: Apache License 2.0
+  license_family: APACHE
+  summary: User-friendly graphical tool for using proteomics identification search engines
+  description: |
     SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.
+
 package:
   name: searchgui
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
 
 source:
-  url: http://genesis.ugent.be/maven2/eu/isas/searchgui/{{ name }}/{{ version }}/{{ name }}-{{ version }}-mac_and_linux.tar.gz
-  sha256: da610fa266a2d30cb3ebfd5794e87d0fe7e2105afcd5d88c6a6576ead6e7073a
+#  url: http://genesis.ugent.be/maven2/eu/isas/searchgui/{{ name }}/{{ version }}/{{ name }}-{{ version }}-mac_and_linux.tar.gz
+  url: https://github.com/CarlosHorro/PeptideShaker2-tests/raw/master/SearchGUI-4.0.1-SNAPSHOT-mac_and_linux.tar.gz
+  sha256: 76c0dc1470c6ff549d14f0b3f4ec19f82e02bbdd80c695873ca09074b0c5f88b
```

Comment:
Has this package moved?

Reply:
This release is a special release not hosted in the same place than usual. Next official releases will be hosted as usual.